x-pack/plugins/security_solution/public/common/containers/use_local_storage.test.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useSecurityLocalStorage, UseSecurityLocalStorage } from './use_local_storage';
+
+const localStorageMock = () => {
```

Comment:
This test file will be refactored when https://github.com/elastic/kibana/pull/67156 is merged.